libraries/ts/src/margin/marginClient.ts

```diff
@@ -148,7 +151,7 @@ export class MarginClient {
           ) {
             token = config.tokens[tokenAbbrev] as MarginTokenConfig
             if (
-              token.symbol === "SOL" &&
+              token.mint === NATIVE_MINT &&
```

Comment:
Can you please review the code style doc. 
https://jetprotocol.atlassian.net/wiki/spaces/JC/pages/32801235/Typescript+Codestyle

Please test your code in a frontend or unit tests before asking for a review

It's not optional. This code won't work because `PublicKey === PublicKey` and `BN === BN` don't do anything

```suggestion
              token.mint.equals(NATIVE_MINT) &&
```

Reply:
token.mint is an address not a PublicKey so i'll have to compare .toString()

Reply:
In that case it can be converted to public key quickly
`translateAddress(token.mint).equals(NATIVE_MINT)`